pages/how-to/deploy/pipelines.md

```diff
@@ -48,7 +48,7 @@ spec:
         tag: latest # VERSION
   clusterRef:
     kind: Cluster
-    name: plrl-how-to-workload-00-prod # replace this with whatever you might have named your dev cluster
+    name: plrl-how-to-workload-00-prod # replace this with whatever you might have named your prod cluster
```

Comment:
think we should make this much more clear because of the immutability of that attribute, make it something like:

```yaml
name: REPLACE_ME_WITH_PROD_CLUSTER_NAME # replace this with whatever you might have named your prod cluster
```

And also add below a:

```
{% callout severity="warn" %}
The `clusterRef` field on a service deployment is immutable.  If you happen to chose the wrong one, it's not a big deal, but you'll need to delete that ServiceDeployment cr manually then let the underlying service recreate it from scratch.  This can be done in the Plural Kubernetes dashboard UI easily.
{% /callout %}
```

Reply:
added